src/vendor_upstream/core/cancelAnimationFramePolyfill.js

```diff
@@ -9,16 +9,18 @@
  * @providesModule cancelAnimationFramePolyfill
  */
 
+import globalThis from './globalThisPolyfill';
+
 /**
  * Here is the native and polyfill version of cancelAnimationFrame.
  * Please don't use it directly and use cancelAnimationFrame module instead.
  */
 const cancelAnimationFrame =
-  global.cancelAnimationFrame ||
-  global.webkitCancelAnimationFrame ||
-  global.mozCancelAnimationFrame ||
-  global.oCancelAnimationFrame ||
-  global.msCancelAnimationFrame ||
-  global.clearTimeout;
+  globalThis.cancelAnimationFrame ||
```

Comment:
From https://github.com/schrodinger/fixed-data-table-2/pull/674, where `global` doesn't exist in vitejs environments, so we instead use a polyfilled version of `globalThis` (which is standard).